src/main/kotlin/snyk/container/KubernetesImageCache.kt

```diff
@@ -53,6 +53,7 @@ class KubernetesImageCache(val project: Project) {
         }
     }
 
+    // todo: recheck logic here, also do we need to handle VFileMoveEvent too?
```

Comment:
Why is this `todo` still there? We consider `move` already, correct?

Reply:
Yeah, the one in L131

Reply:
This PR doesn't address any changes needed for KubernetesImageCache (as well as OSS and SnykCode related changes). So hopefully future me/you will check if any corrections needed here in accordance to the flow described in SnykBulkFileListener kdoc.

Reply:
fixed. Without that KubernetesImageCache  the rest isn't working properly. 
PS: OSS and SnykCode still need to be revised